src/main/webapp/i18n/en/result.json

```
@@ -70,7 +70,8 @@
                     "tooltipOnlineIde": "No result was found / could be generated in time for your most recent submission. Please check your code manually as far as possible."
                 }
             },
-            "notLatestSubmission": "Note: This result and feedback is <strong>not</strong> based on the latest submission."
+            "notLatestSubmission": "Note: This result and feedback is <strong>not</strong> based on the latest submission.",
+            "seeMore": "See more"
```

```suggestion
            "seeMore": "Show more"
```
Intuitively ‘See more’ sounds a bit weird to me, but can’t really explain why. Maybe I see it more like ‘Show more’ as shorthand for ‘Click here to show more information’. That would also align better with the German variant.